platform/openide.util.lookup/apichanges.xml

```diff
@@ -25,6 +25,33 @@
     <apidef name="lookup">Lookup API</apidef>
 </apidefs>
 <changes>
+    <change id="ProxyLookupController">
+        <api name="lookup"/>
+        <summary>Add ProxyLookup.Controller to set lookups dynamically without 
+            subclassing</summary>
+        <version major="8" minor="43"/>
+        <date year="2020" month="7" day="4"/>
+        <author login="tboudreau"/>
+        <compatibility addition="yes" source="compatible" semantic="compatible" binary="compatible"/>
+        <description>
+            <p>
+                One of the most common uses of ProxyLookup is to dynamically 
+                change the set of lookups being delegated to.  The objection
+                to exposing the <code>setLookups()</code> method is straightforward -
+                it would be inviting <i>clients</i> of such a Lookup to call it.
+            </p><p>
+                A cursory grep of NetBeans' own sources finds <b>53 such subclasses</b>
```

Comment:
Rather than describing usage of `grep`, I suggest to make the changes in the overall NetBeans sources, once we agree on the API.